pep-9999.rst

```diff
@@ -85,7 +85,112 @@ Supported Constructs
 ====================
 
 This sections lists constructs that compliant type checkers are expected
```

Comment:
Either in this section or the style guide section, I think it would be worth calling out the difference between an unsupported construct (undefined behavior) and a style guide violation (undesirable but still handled correctly by type checkers).

Reply:
I tried to add a paragraph to the style guide section that tries to clarify this in #34.

Reply:
That looks great, thanks!